extension/persistence/eclipselink/build.gradle.kts

```diff
@@ -37,4 +37,11 @@ dependencies {
   testRuntimeOnly("org.junit.platform:junit-platform-launcher")
 }
 
-sourceSets { test { resources { srcDir("src/test/data") } } }
+tasks.register<Jar>("archiveConf") {
```

Comment:
Tip (not a change request) to avoid string-typo-errors in such cases:

```kolin
val archiveConf by tasks.registering(Jar::class) {
...
}

tasks.named("test") { dependsOn(archiveConf) }
```